logstash-8.yaml

```diff
@@ -368,9 +384,9 @@ subpackages:
       environment:
         contents:
           packages:
-            - wolfi-base
+            - openjdk-17
         environment:
-          LS_JAVA_HOME: /usr/lib/jvm/default-jvm
```

Comment:
`default-jvm` is still valid and should be preferred unless multiple JDKs are used